sdk/storage/storage-blob-changefeed/review/storage-blob-changefeed.api.md

```diff
@@ -18,12 +18,17 @@ export type AccessTier = "P4" | "P6" | "P10" | "P15" | "P20" | "P30" | "P40" | "
 
 // @public
 export class BlobChangeFeedClient {
-    constructor(url: string, credential?: StorageSharedKeyCredential | AnonymousCredential | TokenCredential, options?: StoragePipelineOptions);
+    constructor(url: string, credential?: StorageSharedKeyCredential | AnonymousCredential | TokenCredential, options?: StoragePipelineOptions, changeFeedClientOptions?: BlobChangeFeedClientOptions);
```

Comment:
Could we have `BlobChangeFeedClientOptions` extending `StoragePipelineOptions` so we only have one option bag? `options?: BlobChangeFeedClientOptions`?

Reply:
I looked into .Net's implementation. They are using two separated options: one for Blob client (with pipeline settings), one for change feed.  Here we'll keep consistent with .Net's design for now.